docs/source/model_design/concept_model/ensemble_distributions/index.rst

```diff
@@ -222,3 +221,170 @@ specified.
 
 Sampling from Ensemble Distributions in Vivarium
 ------------------------------------------------
+
+Vivarium needs to sample values of the exposure variable :math:`E` from its
+estimated ensemble distribution in order to assign an exposure value to each
+simulant. This contrasts with the usage of ensemble distributions in GBD, where
+a typical use case might be to estimate the prevalence of each exposure category
+of :math:`E` by computing areas under its PDF. In particular, computing the PDF
+or CDF of :math:`E` is generally sufficient for a GBD team, whereas sampling
+values from the ensemble distribution requires an additional algorithm.
+
+Below, we describe two possible strategies for sampling from an ensemble
+distribution. The first strategy (:ref:`two-step sampling
+<two_step_sampling_of_mixtures>`) requires two random choices for each sampled
+value but is easy to implement if we already have a sampling strategy for all
+the base distributions. The second strategy (:ref:`inverse transform sampling
+<inverse_transform_sampling_of_mixtures>`) uses a single random choice for each
+sampled value but requires calculation of the ensemble distribution's quantile
+function, which is not straightforward. As noted above, a GBD ensemble
+distribution is mathematically defined as a mixture distribution, so it is
+sufficient to describe how to sample from mixture distributions.
```

Comment:
@alibow I updated this introductory paragraph with brief descriptions of the two sampling methods. Does this address your concern about being able to quickly pick up the differences between them?

Reply:
Yes, this looks good to me, thank you!

Reply:
Great, thanks!